pkg/commands/process/settings/rules/ruby/third_parties/bigquery/testdata/datatype_in_table_insert_async.rb

```diff
@@ -0,0 +1,11 @@
+bigquery = Google::Cloud::Bigquery.new
+dataset = bigquery.dataset("my_dataset")
+table = dataset.table("my_table")
+
+inserter = table.insert_async do |result|
+  call
+end
+
+user = { "first_name" => "someone" }
+rows = [user]
+inserter.insert(rows)
```

Comment:
Not a blocker, but is it worth adding some examples with alternative syntax, or is this likely unrealistic? E.g. 

```
inserter.insert([{ first_name: user.first_name }])
```